libmspack4ntest/Basic.cs

```diff
@@ -1,61 +1,77 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Reflection;
+using System.Linq;
```

Comment:
I know this is just tests, but don't see where we are we taking a dependency on Linq?